packages/pigeon/lib/java_generator.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'package:pigeon/functional.dart';
+import 'package:pigeon/pigeon_lib.dart';
```

Comment:
Same here; why are these two package imports?

Reply:
Done. I'm not sure why they are package imports.  I don't know if I wrote that or if it was generated.